app-backend/api/src/main/scala/project/Routes.scala

```diff
@@ -950,11 +950,20 @@ trait ProjectRoutes
         val scenesToIngest = SceneWithRelatedDao.getScenesToIngest(projectId)
         val x: ConnectionIO[List[Scene.WithRelated]] = for {
           _ <- scenesAdded
-          scenes <- scenesToIngest
+          toIngest <- scenesToIngest
```

Comment:
It seems like duplicated logic from here to the end of this for comprehension as seen in `addScenesToProject` function of `ProjectDao`.